client/src/features/Event/New/components/EventCreateForm.tsx

```diff
@@ -199,55 +218,109 @@ export const EventCreateForm = ({ isEdit, eventId }: EventCreateFormProps) => {
                 isRequired
               />
             </Flex>
+          </Flex>
 
-            <Input
-              id="registrationEnd"
-              name="registrationEnd"
-              label="신청 종료일"
-              type="datetime-local"
-              placeholder="2025.07.25 15:00"
-              value={basicEventForm.registrationEnd}
-              max={basicEventForm.eventStart}
-              onChange={handleChange}
-              errorMessage={errors.registrationEnd}
-              isRequired
-            />
+          <Flex
+            gap="16px"
+            css={css`
+              @media (max-width: 768px) {
+                flex-direction: column;
+                gap: 30px;
+              }
+            `}
+          >
+            <Flex
+              dir="column"
+              gap="8px"
+              css={css`
+                flex: 1;
+              `}
+            >
+              <label htmlFor="registrationEnd">
+                <Text type="Body">신청 종료일</Text>
+              </label>
```

Comment:
p1.
다음처럼 사용 가능할 것 같아요~!
```suggestion
    <Text as="label" htmlFor="registrationEnd" type="Body">신청 종료일</Text>
```

Reply:
처음엔 저도 요렇게 써봤었는데 label 클릭했을 때 input에 포커스가 가지 않아서 현재와 같은 방식으로 변경하였습니당 ,,